applications/nrf5340_audio/dfu/conf/overlay-dfu_external_flash.overlay

```diff
@@ -9,18 +9,12 @@
 };
 
 &spi4 {
-	cs-gpios = <&gpio0 11 GPIO_ACTIVE_LOW>, <&gpio0 17 GPIO_ACTIVE_LOW>, < &gpio1 10 GPIO_ACTIVE_LOW >;
+	cs-gpios = <&gpio1 10 GPIO_ACTIVE_LOW>, <&gpio0 11 GPIO_ACTIVE_LOW>, <&gpio0 17 GPIO_ACTIVE_LOW>;
 	status = "okay";
-	sdhc0: sdhc@0 {
-		status = "okay";
-	};
-	cs47l63: cs47l63@1 {
-		status = "okay";
-	};
-	mx25r64: mx25r6435f@2 {
+	mx25r64: mx25r6435f@0 {
 
 		compatible = "jedec,spi-nor";
-		reg = <2>;
+		reg = <0>;
```

Comment:
What happened here?

Reply:
This value has to be the same slot as the node name says (in this case `mx25r6435f@0`), otherwise a warning will be displayed. Not sure why the `reg` property is needed in addition to the node tag